core/src/com/unciv/models/ruleset/unit/BaseUnit.kt

```diff
@@ -597,21 +597,29 @@ class BaseUnit : RulesetObject(), INonPerpetualConstruction {
 
             when {
                 unique.isOfType(UniqueType.Strength) && unique.params[0].toInt() > 0 -> {
-                    if (unique.conditionals.any { it.isOfType(UniqueType.ConditionalVsCity) })
-                        power += (power * unique.params[0].toInt()) / 200
-                    else if (unique.conditionals.any { it.isOfType(UniqueType.ConditionalVsUnits) })
-                        power += (power * unique.params[0].toInt()) / 400
+                    when {
+                        unique.conditionals.any { it.isOfType(UniqueType.ConditionalVsUnits) } ->
```

Comment:
Sounds like only the vs city is the odd one out
So if else

Reply:
I'd foremost unify the uniques and promotions loop. Why should 'May Paradrop' have a value to the AI when it's on a unique but not then it's on a promotion? The other differences in these loops are academic, too. Pity the duplication of `power += (power * unique.params[0].toInt()) /` can't be trivially simplified.